netlify.toml

```
@@ -0,0 +1,24 @@
+# Netlify build instructions
+[build]
+    command = "./docs/book/install-and-build.sh"
+    publish = "docs/book/book"
+
+# Standard Netlify redirects
+[[redirects]]
+    from = "https://master--kubernetes-sigs-secrets-store-csi-driver.netlify.com/*"
+    to = "https://master.secrets-store-csi-driver.sigs.k8s.io/:splat"
+    status = 301
+    force = true
+
+# HTTP-to-HTTPS rules
+[[redirects]]
+    from = "http://master.secrets-store-csi-driver.sigs.k8s.io/*"
+    to = "https://master.secrets-store-csi-driver.sigs.k8s.io/:splat"
+    status = 301
+    force = true
+
+[[redirects]]
+    from = "http://master--kubernetes-sigs-secrets-store-csi-driver.netlify.com/*"
```

```suggestion
    from = "http://master--kubernetes-sigs-secrets-store-csi-driver.netlify.app/*"
```
Same as above, netlify no longer uses .com for hosted sites. It's now .app

---

Updated!